src/main/osd/osd_elements.c

```diff
@@ -149,6 +149,7 @@
 
 #include "flight/gps_rescue.h"
 #include "flight/position.h"
+#include "flight/position_control.h"
```

Comment:
What is this for? Is osd_elements.c dependent on something in position_control.h?

Reply:
Removed